pkg/plugins/runtime/universal/outbound/outbound.go

```diff
@@ -0,0 +1,112 @@
+package outbound
+
+import (
+	"context"
+	"time"
+
+	"github.com/kumahq/kuma/pkg/core"
+	"github.com/kumahq/kuma/pkg/dns/resolver"
+
+	"github.com/golang/protobuf/proto"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+	"github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
+	"github.com/kumahq/kuma/pkg/core/resources/manager"
+	"github.com/kumahq/kuma/pkg/core/resources/store"
+	"github.com/kumahq/kuma/pkg/dns"
+)
+
+var log = core.Log.WithName("dns-vips-allocator")
+
+type OutboundsLoop struct {
+	rorm      manager.ReadOnlyResourceManager
+	rm        manager.ResourceManager
+	resolver  resolver.DNSResolver
+	newTicker func() *time.Ticker
+}
+
+const (
+	tickInterval = 500 * time.Millisecond
```

Comment:
That cannot be hardcoded IMHO. I think the easiest way here is to just take a value from XDS Interval. There is no point in updating every 0,5s when the XDS interval is for example 10s.